packages/circuit-ui/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # @sumup/circuit-ui
 
+## 4.16.0
+
+### Minor Changes
+
+- [#1401](https://github.com/sumup-oss/circuit-ui/pull/1401) [`d8361322`](https://github.com/sumup-oss/circuit-ui/commit/d836132205fd376ff9c02c414fb0e69cb73a654e) Thanks [@hris27](https://github.com/hris27)! - Added `useSidePanel` hook and `SidePanelProvider`.
+
+* [#1401](https://github.com/sumup-oss/circuit-ui/pull/1401) [`d8361322`](https://github.com/sumup-oss/circuit-ui/commit/d836132205fd376ff9c02c414fb0e69cb73a654e) Thanks [@hris27](https://github.com/hris27)! - Added `update` operation to the `useStack` hook.
+
+### Patch Changes
+
+- [#1401](https://github.com/sumup-oss/circuit-ui/pull/1401) [`d8361322`](https://github.com/sumup-oss/circuit-ui/commit/d836132205fd376ff9c02c414fb0e69cb73a654e) Thanks [@hris27](https://github.com/hris27)! - Changed `Popover` strategy to `fixed` to prevent overflow issues.
```

Comment:
```suggestion
- [#1401](https://github.com/sumup-oss/circuit-ui/pull/1401) [`d8361322`](https://github.com/sumup-oss/circuit-ui/commit/d836132205fd376ff9c02c414fb0e69cb73a654e) Thanks [@hris27](https://github.com/hris27)! - Changed the `Popover`'s strategy to `fixed` to prevent overflow issues.
```